packages/wonder-blocks-data/components/data.test.js

```diff
@@ -457,6 +457,36 @@ describe("Data", () => {
                     expect(fulfillRequestFn).toHaveBeenCalledTimes(1);
                     expect(fakeHandler.fulfillRequest).not.toHaveBeenCalled();
                 });
+
+                it("should invoke handler if interceptor returns null", () => {
+                    // Arrange
+                    const fakeHandler: IRequestHandler<string, string> = {
+                        fulfillRequest: jest.fn(() => Promise.resolve("data")),
+                        getKey: (o) => o,
+                        shouldRefreshCache: () => false,
+                        type: "MY_HANDLER",
+                        cache: null,
+                    };
+                    const fakeChildrenFn = jest.fn(() => null);
+                    const fulfillRequestFn = jest.fn(() => null);
```

Comment:
Is `fulfillRequestIFn` the interceptor?  If so, rewording the test title to something like the following may be clearer:
> should invoke handler's fulfillRequest if InterceptData's returns null

Thoughts?